jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/audit/ServicesAwareAuditEventBuilder.java

```diff
@@ -13,72 +13,77 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 package org.jbpm.kie.services.impl.audit;
 
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
 import org.jbpm.process.audit.NodeInstanceLog;
 import org.jbpm.process.audit.ProcessInstanceLog;
 import org.jbpm.process.audit.VariableInstanceLog;
 import org.jbpm.process.audit.event.AuditEvent;
 import org.jbpm.process.audit.event.DefaultAuditEventBuilderImpl;
+import org.jbpm.process.core.context.variable.VariableScope;
+import org.jbpm.process.instance.ProcessInstance;
+import org.jbpm.process.instance.context.variable.VariableScopeInstance;
 import org.kie.api.event.process.ProcessCompletedEvent;
 import org.kie.api.event.process.ProcessNodeLeftEvent;
 import org.kie.api.event.process.ProcessNodeTriggeredEvent;
 import org.kie.api.event.process.ProcessStartedEvent;
 import org.kie.api.event.process.ProcessVariableChangedEvent;
 import org.kie.internal.identity.IdentityProvider;
-
+import org.kie.server.api.KieServerConstants;
 
 public class ServicesAwareAuditEventBuilder extends DefaultAuditEventBuilderImpl {
 
-    private IdentityProvider identityProvider;    
-    
+    private IdentityProvider identityProvider;
+
     private String deploymentUnitId;
 
+    private final Boolean allowSetInitiator = Boolean.parseBoolean(System.getProperty(KieServerConstants.CFG_BYPASS_AUTH_USER, "false"));
```

Comment:
just copy the name instead of referring to kieserverconstants

Reply:
Change done.
Using property name instead.